js/apps/account-ui/pom.xml

```
@@ -138,6 +138,11 @@
                             <token><![CDATA[</head>]]></token>
                             <value xml:space="preserve">
 <![CDATA[
+    <#if properties.scripts?has_content>
+      <#list properties.scripts?split(' ') as script>
+      <script type="text/javascript" src="${resourceUrl}/${script}"></script>
```

We can omit the `type="text/javascript", this is already the default for HTML5 documents.
```suggestion
        <script src="${resourceUrl}/${script}"></script>
```